.travis.yml

```diff
@@ -1,28 +1,47 @@
 language: php
 
+dist: xenial
+
+services:
+  - mysql
+  - postgresql
+  - xvfb
+
+addons:
+  apt:
+    packages:
+      - tidy
+      - chromium-chromedriver
+      - chromium-browser
+
 env:
   global:
     - COMPOSER_ROOT_VERSION=2.0.x-dev
 
 matrix:
   include:
-    - php: 5.6
-      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
-    - php: 7.0
-      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=PGSQL PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_COVERAGE_TEST=1
+    - php: 7.1
+      env: DB=PGSQL PHPUNIT_TEST=1
+    - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1
+    - php: 7.3
+      env: DB=MYSQL PHPUNIT_TEST=1
 
 before_script:
+  - sudo apt-get remove -y --purge google-chrome-stable || true
+
   # Init PHP
   - phpenv rehash
   - phpenv config-rm xdebug.ini
 
   # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/recipe-core:1.0.x-dev
-  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql 2.0.x-dev; fi
-  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - composer require --no-update silverstripe/recipe-core:^4
```

Comment:
We tend to target the dev branch of recipe-core/recipe-cms in our tests ... although in this case it probably won't make much difference.

Reply:
yeah, dev branch here would increase chances of false-positives though. not sure if there's actual value in that.